SECURITY.md

```diff
@@ -6,25 +6,41 @@ This is the security notice for all GDS (Government Digital Service) repositorie
 
 ## Reporting a Vulnerability
 
-GDS is an advocate of responsible vulnerability disclosure. If you’ve found a vulnerability, we would like to know so we can fix it.
+GDS is an advocate of responsible vulnerability disclosure. If you’ve found a vulnerability, we would like to know so we can fix it. Quick links can be found at our [security.txt](https://vdp.cabinetoffice.gov.uk/.well-known/security.txt).
 
-Send details to [disclosure@digital.cabinet-office.gov.uk], including:
+You can [report a vulnerability to GDS using HackerOne](https://hackerone.com/44c348eb-e030-4273-b445-d4a2f6f83ba8/embedded_submissions/new), please include:
 - the website, page or repository where the vulnerability can be observed
 - a brief description of the vulnerability 
+- details of the steps we need to take to reproduce the vulnerability
 - optionally the type of vulnerability and any related [OWASP category]
```

Comment:
Suggested wording tweak:

`the type of vulnerability, for example, the [OWASP category](https://owasp.org/www-community/vulnerabilities/)`

Also suggest we move "any related" into a separate line?